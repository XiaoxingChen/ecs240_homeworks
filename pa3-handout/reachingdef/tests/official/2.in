c Straightline code
p 2 6 5 1 6
c bb1: v1 = c;
b 1 1
e 1 2
c bb2: v2 = c;
b 2 2
e 2 3
c bb3: v1 = v1 + v2;
b 3 1 1 2
e 3 4
c bb4: ;
b 4
e 4 5
c bb5: print(v1, v2);
b 5 0 1 2
e 5 6
c bb6: v2 = v2 + v1;
b 6 2 2 1
