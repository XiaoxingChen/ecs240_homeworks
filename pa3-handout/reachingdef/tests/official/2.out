rdout 1 1
rdout 2 1 2
rdout 3 2 3
rdout 4 2 3
rdout 5 2 3
rdout 6 3 6
